cs/cs/VowpalWabbitMultiLineExampleCollection.cs

```diff
@@ -90,7 +93,9 @@ private TPrediction Execute<TPrediction>(VowpalWabbit vw, Action<VowpalWabbitExa
                 // signal end-of-block
                 empty = vw.GetOrCreateNativeExample();
                 empty.MakeEmpty(vw);
-                predictOrLearn(empty);
+                ec_col.Add(empty);
```

Comment:
is the newline example still needed at the end of the collection?

Reply:
I'd hope not---we shouldn't need two delimiters.

Reply:
Empty line was used to trigger multiline prediction in reductions before.  Not needed to trigger multiline learn/predict in reductions anymore.  However some reductions used to add the emptyline before processing.  I left this here to be backward compatible.